File: Javascript/Bubblesort.js
let bubbleSort = (unordered) => {
  let flag;
  do {
    flag = false;
    for (let i = 0; i < unordered.length; i++) {
      if (unordered[i] > unordered[i + 1]) {
        let aux = unordered[i];
        unordered[i] = unordered[i + 1];
        unordered[i + 1] = aux;
        flag = true;
      }
    }
  } while (flag);
  return unordered;
};

const unsorted = [9, 5, 1, 2, 7, 10, 0, -1];
const sorted = bubbleSort(unsorted);
console.log(sorted);

File: Python/BubbleSort.py
def bubbleSort(arr):
  for i in range(0,len(arr)-1):
    sorted=True                     
    for j in range(0,len(arr)-1-i): 
      if arr[j]>arr[j+1]:
        arr[j],arr[j+1]=arr[j+1],arr[j]
        sorted=False
    
    if sorted:
      break    
      
  return arr      

arr=[int(x) for x in input().split()]
bubbleSort(arr)
print(arr)
